### ISSUE TITLE
New release planned?

### ISSUE THREAD

**Issue by patlecat** [ThePhD/sol2#971]
Do you have any bug fix release planned and any feature release this year? Is there maybe a roadmap? I believe Lua5.4 is coming soon too btw.

Would be great to know in advance a bit so we can plan too.

**Comment by ThePhD** [comment]
Yes, I am planning a new release. I am hoping to do it later in June or, depending on how speaking at C++ on Sea and C++ Russia goes, by the end of August accounting for all potential buses I can be hit by.

I have got a lot on my plate, but the new release should be backwards compatible and just fix bugs, so nothing to really worry about. I've been testing against latest Lua 5.4 as well, even if the CI is in a perpetual state of red because TravisCI and Appveyor are the bane of my existence.

(Part of the new release is moving everything to GitHub actions, but my CI builds are... pretty involved, unfortunately. 😭)

**Comment by fndisme** [comment]
what's new in 3.2.0? I saw the new release , but not the comment. Bug fixed or new feather?

**Comment by TerensTare** [comment]
Any plans on replacing custom type_traits with C++20 concepts on a near future? I think they are a worthy because AFAIK they can reduce compile times. (of course, backwards compatibility is important as well)

**Comment by ThePhD** [comment]
That'd bump the requirements up to C++20 and people are already kind of miffed about the C++17 requirement.

On the other hand, I actually need to change a lot of the templates to use other techniques to reduce compile times, because even if I did a big cleanup for `if constexpr` I have a LOT of other garbage to clean up...

**Comment by ThePhD** [comment]
Temporary release to get Visual Studio off everyone's back: https://github.com/ThePhD/sol2/releases/tag/v3.2.1

**Comment by ThePhD** [comment]
(This release should also be fine with Lua 5.4, modulo an explosion.)

**Comment by TerensTare** [comment]
Thank you @ThePhD ! I'll try it immediately.

**Comment by patlecat** [comment]
> 
> 
> New release planned eventually.
Kool and what's planned in it? And when could it happen?
